dockerfile/sdk/local.Dockerfile

```diff
@@ -0,0 +1,110 @@
+FROM golang:1.19-alpine3.16 AS build-env
+
+RUN apk add --update --no-cache curl make git libc-dev bash gcc linux-headers eudev-dev ncurses-dev
+
+ARG TARGETARCH
+ARG BUILDARCH
+ARG GITHUB_ORGANIZATION
+ARG REPO_HOST
+ARG GITHUB_REPO
+
+WORKDIR /go/src/${REPO_HOST}/${GITHUB_ORGANIZATION}/${GITHUB_REPO}
+
+ADD . .
```

Comment:
This Dockerfile is the same as native.Dockerfile except for this part

Reply:
This info be really useful as a comment in this file. 😄

Reply:
good call, done